.storybook/preview.js

```diff
@@ -6,7 +6,7 @@ import { GlobalStyle } from '../src/components/layout/global-style';
 addDecorator((storyFn) => (
   <>
     <ThemeProvider theme={theme}>
-      <GlobalStyle />
+      <GlobalStyle $lightTheme={true} />
```

Comment:
How to access the dark theme then? There is a background toggle in storybook (dark & light), is there a way of toggling our theme in general? Because we will need to check our components on both backgrounds for sure.

What design is more correct right now to begin with? Light? Than this is fine, otherwise we need to rollback.

Reply:
We agreed to focus on the light theme for the beginning. To make things easer I suggested to globally turn on the light theme here in the `preview.js`. We can think about this and figure out how to make use of the Storybook-internal Theme switcher in the future. Let's first get the light theme looking splendid :sparkles: